### DISCUSSION TITLE
Missing commit in release 1.29.3

### DISCUSSION THREAD

**Discussion by chrisjbrown** [ueberdosis/tiptap#792] [General Questions] [answered]
In dist/tiptap.esm.js I'm expecting to see
`var element = parser.parseFromString(htmlString, 'text/html').body.firstElementChild`

but instead see this
`var element = parser.parseFromString(htmlString, 'text/html').body;`

the line is correct in `src/Editor.js`
`const element = parser.parseFromString(htmlString, 'text/html').body.firstElementChild`

have changes been made to the release process?

![image](https://user-images.githubusercontent.com/1250638/89891320-28f6c780-dbd5-11ea-90cd-ca0101564606.png)

**Comment by hanspagel** [comment, marked as answer]
Thanks for the report! Fixed thanks to your bug report in #797. 🙌

Everyone landing here: Update to the latest version.
